linebot/exceptions.py

```diff
@@ -58,22 +58,24 @@ def __init__(self, message='-'):
 class LineBotApiError(BaseError):
     """When LINE Messaging API response error, this error will be raised."""
 
-    def __init__(self, status_code, error=None):
+    def __init__(self, status_code, request_id=None, error=None):
         """__init__ method.
 
         :param int status_code: http status code
+        :param str request_id: Request ID. A unique ID is generated for each request
```

Comment:
[ask]
It seems that request_id is optional, How about adding `(optional)` to description?

Reply:
This exception is thrown when Messaging API returns 4xx, 5xx error. And Messaging API always returns `request_id` in response headers. So, basically `LineBotApiError` has `request_id`.

Which do you think is better?
- Both `request_id` and `error` is `(optional)`
- Both `request_id` and `error` is required

Reply:
This exception is thrown when Messaging API returns 4xx, 5xx error. And Messaging API always returns `request_id` in response headers. So, basically `LineBotApiError` has `request_id`.

Which do you think is better?
- Both `request_id` and `error` is `(optional)`
- Both `request_id` and `error` is required

Reply:
I think that the former is better for the irregular case (e.g. response does not contain `request_id` and `error` for some reason)